src/lib/mina_base/transaction_status.ml

```diff
@@ -47,6 +47,28 @@ module Failure = struct
     end
   end]
 
+  module Table = struct
```

Comment:
Nit: `Table` normally refers to the `Core` hash table type. Could you rename this? (`Collection`, perhaps?)